org.eclipse.lsp4xml.extensions.maven/src/main/java/org/eclipse/lsp4xml/extensions/maven/MavenProjectCache.java

```diff
@@ -122,6 +122,12 @@ private void parse(DOMDocument document) {
 				}
 			} else {
 				e.getResults().stream().flatMap(result -> result.getProblems().stream()).forEach(problems::add);
+				if (e.getResults().size() == 1) {
```

Comment:
To make sure I understand, is the size of `getResults()` only 1 when an incomplete pom model error occurs?

Reply:
Yes, but in my tests, size is always 1